packages/components/src/popover/index.js

```diff
@@ -87,6 +87,15 @@ class Popover extends Component {
 		if ( prevProps.position !== this.props.position ) {
 			this.computePopoverPosition( this.state.popoverSize, this.anchorRect );
 		}
+
+		if ( prevProps.anchorRect !== this.props.anchorRect ) {
+			this.refreshOnAnchorMove();
+		}
+
+		const hasAnchorRect = this.props.hasOwnProperty( 'anchorRect' );
+		if ( hasAnchorRect !== prevProps.hasOwnProperty( 'anchorRect' ) ) {
```

Comment:
Why use `hasOwnProperty`?

Reply:
> Why use `hasOwnProperty`?

Because we expect the value could change over time, but scheduling the interval doesn't depend on the value, it just depends on whether a value _exists_.